torch/utils/benchmark/utils/valgrind_wrapper/timer_interface.py

```diff
@@ -91,13 +101,20 @@ def __sub__(
         return self._merge(other, lambda c: -c)
 
     def transform(self, map_fn: Callable[[str], str]) -> "FunctionCounts":
+        """Apply `map_fn` to all of the function names.
+
+        This can be used to regularize function names (e.g. stripping irrelevant
+        parts of the file path), coalesce entries by mapping multiple functions
+        to the same name, etc.
+        """
         counts: DefaultDict[str, int] = collections.defaultdict(int)
         for c, fn in self._data:
             counts[map_fn(fn)] += c
 
         return self._from_dict(counts, self.inclusive)
 
     def filter(self, filter_fn: Callable[[str], bool]) -> "FunctionCounts":
+        """Keep only the elements where `filter_fn` returns True."""
```

Comment:
where `filter_fn` applied to function name returns True

Reply:
Done.